packages/store/src/internal/internals.ts

```diff
@@ -65,6 +65,13 @@ export interface MappedStore {
   depth: string;
 }
 
+export type StatesAndDefaults =
+  | {
+      defaults: any;
+      states: MappedStore[];
+    }
+  | undefined;
```

Comment:
Why is `undefined` a value in the type, surely specifying the option of undefined at the argument would be a clearer way to express this?
ie. (in lifecycke-state-manager.ts)
`ngxsBootstrap<T>(action: T, results: StatesAndDefaults | undefined): void {`